packages/dds/tree/src/shared-tree/schematizeTree.ts

```diff
@@ -221,7 +221,7 @@ export function initialize(checkout: ITreeCheckout, treeContent: TreeStoredConte
  * @returns true iff checkout now is compatible with `viewSchema`.
  */
 export function ensureSchema(
-	viewSchema: ViewSchema,
+	viewSchema: SchemaCompatibilityTester,
```

Comment:
Should we rename the variable as well?

Reply:
(Same question for other instances)

Reply:
I think it's still semantically valid, as this is carrying the information about the view schema.